lib/rails_response_dumper/option_parser.rb

```diff
@@ -7,6 +7,10 @@ def self.parse_options!
     options = {}
 
     OptionParser.new do |opts|
+      opts.on('--dumps-dir DIRECTORY_NAME', 'Output dumps to this directory.') do |v|
```

Comment:
If we continue to use rspec as a basis for documentation, then this should be PATH:

```suggestion
      opts.on('--dumps-dir PATH', 'Output dumps to this directory.') do |v|
```